gdk4/src/auto/toplevel.rs

```diff
@@ -50,15 +49,15 @@ pub trait ToplevelExt: 'static {
 
     fn focus(&self, timestamp: u32);
 
-    fn get_state(&self) -> SurfaceState;
+    //fn get_state(&self) -> /*Ignored*/ToplevelState;
```

Comment:
This should be easy to generate by simply putting the enum into the generate list :)